apps/web/modules/survey/editor/components/matrix-question-form.test.tsx

```
@@ -120,6 +146,25 @@ vi.mock("@/modules/ui/components/shuffle-option-select", () => ({
+// Mock MatrixSortableItem component
+vi.mock("@/modules/survey/editor/components/matrix-sortable-item", () => ({
+  MatrixSortableItem: vi.fn(({ choice, index, type, onDelete, onKeyDown, canDelete }) => (
+    <div data-testid={`matrix-sortable-item-${type}-${index}`}>
+      <div data-testid={`grip-${type}-${index}`} />
+      <input
+        data-testid={`input-${type}-${index}`}
+        defaultValue={choice?.label?.default || ""}
+        onKeyDown={onKeyDown}
+      />
+      {canDelete && (
+        <button data-testid={`delete-${type}-${index}`} onClick={() => onDelete(index)}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)